_data/negotiation-mastery-program.yaml

```diff
@@ -0,0 +1,174 @@
+title: Negotiation Mastery Program
+mission: Awaken the negotiator within—calm, clear, and creative.
+principles:
+  - 1st: Relax completely
+  - 2nd: Feel into emotional waves
+  - 3rd: Release tension inviting clarity
+  - 4th: Form reflection into reflex
+community:
+  - url: https://dojo.center/community
+practices:
+  - reflection:
+    - continuous:
+      - Prior to entering an exchange with probability of high tension and or friction.
+      - Leaving an exchange which experienced high tension and or friction.
+    - belt:
+      - In the future as a black belt, how will I show up and what sorts of things do I do?
+      - To this point, what have I learned so far overall and specifically to this belt level?
+      - How do I intend on applying what I have learned in the short and medium term?
+    - systems thinking:
+    - intentional conversation:
+    - challenge:
+      - Every level has challenges to pass in order to prove sufficient learning and skill to proceed.
+    - run the gauntlet:
+      - Before the level's test, one must demonstrate mastery of the previous levels.
+    - forms:
+      - Negotiation 9 Moves (Dynamic)
+      - Zen Leader Essential Flips (Static)
+awareness:
+  - Emotional
+  - Social
+  - Relational
+  - Energetic
+  - Somatic
+  - Metaphorical
+belt level:

```

Comment:
change this to "levels" ... also use underscores for any labels in YAML not spaces between